components/landing/hero.tsx

```diff
@@ -1,3 +1,34 @@
+import { Button } from '../ui';
+import Link from 'next/link';
+
 export default function Hero() {
-  return <section>{/* put the title and subheading here */}</section>;
+  return (
+    <section>
+      {/* Hero Section Container */}
+      <div className="max-w-full h-[70vh] lg:min-h-screen flex flex-col justify-center items-center gap-6 md:gap-8">
```

Comment:
- remove the `max-w-full` cause that is the default for a block element
- just use `h-screen` instead of `min-h-screen` for large screen

```suggestion
      <div className="h-[70vh] lg:h-screen flex flex-col justify-center items-center gap-6 md:gap-8">
```